src/apps/explorer/components/SummaryCardsWidget/VolumeChart/volumeData.json

```diff
@@ -0,0 +1,1038 @@
+[
```

Comment:
Is this mock data still used, now that you have connected to the real data endpoint?

Reply:
Only on storybook for now

Reply:
Ah ok, makes sense then